schema.sql

```diff
@@ -0,0 +1,47 @@
+-- Add your create table query as per instruction.
```

Comment:
- [ ] Kindly add indexes for all foreign keys to speed up the retrieval of data from tables.

Reply:
Hi @gemmen29 ,
Thanks for review. :smile: 
We have added indexes for foreign key.